TeamCode/src/main/java/org/firstinspires/ftc/teamcode/actuators/TankDrive.java

```diff
@@ -0,0 +1,36 @@
+package org.firstinspires.ftc.teamcode.actuators;
+
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.util.Range;
+
+/**
+ * Created by LBYPatrick on 10/27/2017.
+ */
+
+final public class TankDrive {
+    private DcMotor LM = null;
+    private DcMotor RM = null;
+    public double leftPower;
+    public double rightPower;
+
+    public double speed = 0.1;
+
+    public TankDrive(DcMotor leftMotor, DcMotor rightMotor){
```

Comment:
Careful about the possibility of 4 motors

Reply:
It is gonna be 4 motors :)

Reply:
Keyword: TankDrive. I guess tank drive does not need four motors...It's better to write another class called TankDrive_4WD if needed

Reply:
Reason : move() is a function that **ONLY** works for two-wheel drive. Making this class to support both 2WD and 4WD would just make the code messy. (A mess has been made in the ServoControl class, but since I need to use it more than once for different Servos, I have to make it compatible to all cases)

Reply:
Please make a better organization while make sure 2x WD cases works.

Reply:
@gordon-zhao Fixed.